examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+                    #print("env.tick: ", env.tick, " holding: ", holding, " cost: ", cost, "total_money:", total_money, "remaining_money", remaining_money)
+
+                    if holding > 0:  # sub_engine_name -> market
+                        # action = MarketOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick)
+
+                        # limit_order
+                        # action = LimitOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, limit=highest_price)
+
+                        # stop order
+                        # action = StopOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, stop=lowest_price)
+
+                        # stop limit order
+                        # action = StopLimitOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, stop=highest_price, limit=lowest_price)
+
+                        # order that has life_time, default life_time is 1, if life_time > 1, order will keep live when not triggered in life_time ticks
+                        action = LimitOrder(item=decision_event.item, amount=holding,
+                                        direction=OrderDirection.sell, tick=env.tick, limit=highest_price, life_time=10)
+                    else:
+
+                        action = MarketOrder(item=decision_event.item, amount=1000,
+                                        direction=OrderDirection.buy, tick=env.tick)
+                    print(f"    >>>>trade order: {action.id},tick:{env.tick}")
+
+                elif decision_event.action_type == ActionType.cancel_order:
+                    # cancel order decision
+                    print(f"<<<<    Cancel order decision_event:{[x.id for x in decision_event.action_scope.available_orders]},tick:{env.tick}")
+                    if len(decision_event.action_scope.available_orders) > 0:
+                        for i in range(len(decision_event.action_scope.available_orders)):
+                            if random.random() > 0.75:
+                                action = CancelOrder(action=decision_event.action_scope.available_orders[i],
+                                                tick=env.tick)
+                                print(f"    >>>>Cancel order:{action.action.id},tick:{env.tick}")
+                                actions.append(action)
+                    action = None
+                actions.append(action)
+
+    #env.reset()
+    ep_time = time.time() - ep_start
+
+stock_snapshots = env.snapshot_list['stocks']
+
+# query account_hold_num for specified markset
+sz_account_hold_num = stock_snapshots[::"account_hold_num"].reshape(-1, 5)
+
+print("volume holding of sz market.")
```

Comment:
fellow it: https://github.com/microsoft/maro/wiki/Code-Style-Guide
e.g. "Volume ..."